src/debug/ee/debugger.cpp

```diff
@@ -15136,7 +15136,8 @@ HRESULT Debugger::InitAppDomainIPC(void)
     } hEnsureCleanup(this);
 
     DWORD dwStrLen = 0;
-    WCHAR szExeName[MAX_LONGPATH];
+    PathString szExeNamePathString;
+    WCHAR * szExeName = szExeNamePathString.OpenUnicodeBuffer(MAX_LONGPATH);
```

Comment:
Note that it would be better to just use regular SString here that always allocates on heap. The PathString stack-allocated buffer will never kick in here because of we always allocate MAX_LONGPATH.